stages/org.osbuild.systemd.preset

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+"""
+Configure Systemd services through presets.
+
+Enable or disable systemd services through presets.
+"""
+
+import sys
+
+import osbuild.api
+
+SCHEMA = r"""
+"additionalProperties": false,
+"properties": {
+  "presets": {
+    "type": "array",
```

Comment:
nitpick: it would probably make sense to enforce that at least one item is specified in the schema. Specifying no items would be currently a noop, but it could be also hiding a wrong use of the stage by higher-level tools.

Reply:
Ack.